### ISSUE TITLE
spring-cloud-starter-consul-all disables functional binding when using with spring cloud stream

### ISSUE THREAD

**Issue by edwardzjl** [spring-cloud/spring-cloud-consul#654]
Version: Hoxton.SR6

When I include spring-cloud-starter-consul-all in my pom, I got a "Functional binding is disabled due to the presense of @EnableBinding annotation in your configuration" message every time I start the application.

I don't have @EnableBinding anywhere in my program, and if I remove the dependency of spring-cloud-starter-consul-all, this message disappears

The sample code can be viewed [here](https://github.com/edwardzjl/mq-demo)

**Comment by ryanjbaxter** [comment]
Does it happen when using Hoxton.SR5?

cc @olegz

**Comment by edwardzjl** [comment]
> Does it happen when using Hoxton.SR5?
> 
> cc @olegz

Yes it still happens.
I find this annotation in spring-cloud-starter-consul-bus, after I exclude it like

```xml
        <!-- https://mvnrepository.com/artifact/org.springframework.cloud/spring-cloud-starter-consul-all -->
        <dependency>
            <groupId>org.springframework.cloud</groupId>
            <artifactId>spring-cloud-starter-consul-all</artifactId>
            <exclusions>
                <exclusion>
                    <groupId>org.springframework.cloud</groupId>
                    <artifactId>spring-cloud-starter-consul-bus</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
```

The warning disappears.

**Comment by OlgaMaciaszek** [comment]
Yes, it's an issue in Spring Cloud Bus. It [has been addressed](https://github.com/spring-cloud/spring-cloud-bus/commit/17c16bc345feb9097b71db543b1ea33dc138eb71) in the `2020.0.0` release train.
